openbas-api/src/test/java/io/openbas/rest/PayloadApiSearchTest.java

```diff
@@ -204,7 +204,7 @@ void given_filter_input_by_source_should_return_a_page_of_payloads_filter_by_sou
                     .contentType(MediaType.APPLICATION_JSON)
                     .content(asJsonString(searchPaginationInput)))
             .andExpect(status().is2xxSuccessful())
-            .andExpect(jsonPath("$.numberOfElements").value(1));
+            .andExpect(jsonPath("$.numberOfElements").value(3));
```

Comment:
**question:**

Why this changes ?

Reply:
Because, in the beforeAll(), we create 3 payloads: dns(all_arch), command(all_arc), and executable(arm64). The all_arch payload is considered to include both arm64 and x86_64. So, when we search for a payload with this architecture, we now get arm64 along with all_arch.